test/lib/rules/yield-effects-spec.js

```diff
@@ -25,9 +79,73 @@ ruleTester.run("yield-effects", rule, {
   ],
   invalid: [
     {
+      code: "import { take } from 'redux-saga';"
+          + "export default function* saga() {"
+          + "  const sagaTakeEffect = take();"
+          + "  yield sagaTakeEffect;"
+          + "}",
+      errors: [ERRORS.noyield]
+    },
+    {
+      code: "import { take as sagaTake } from 'redux-saga';"
+          + "export default function* saga() {"
+          + "  const sagaTakeEffect = sagaTake();"
+          + "  yield sagaTakeEffect;"
```

Comment:
Sorry for the delay. 

I would test for the patterns described in the redux-saga documentation, starting from here: http://yelouafi.github.io/redux-saga/docs/basics/UsingSagaHelpers.html and here: http://yelouafi.github.io/redux-saga/docs/advanced/RunningTasksInParallel.html

Note that the effect creators are imported from `redux-saga/effects`, but the helpers which are to be `yield*`ed, not `yield`ed, are imported from `redux-saga` itself:
```
import { call, put } from 'redux-saga/effects'
import { takeEvery } from 'redux-saga'
```

Note that `yield`ing an array of calls to effect creators is also valid:
```
// correct, effects will get executed in parallel
const [users, repos]  = yield [
  call(fetch, '/users'),
  call(fetch, '/repos')
]
```